collect_app/src/androidTest/java/org/odk/collect/android/support/ResetStateRule.java

```diff
@@ -88,11 +89,11 @@ private void resetDagger() {
         }
     }
 
-    private void clearPrefs() {
-        settingsProvider.getGeneralSettings().clear();
-        settingsProvider.getGeneralSettings().setDefaultForAllSettingsWithoutValues();
-        settingsProvider.getAdminSettings().clear();
-        settingsProvider.getAdminSettings().setDefaultForAllSettingsWithoutValues();
-        settingsProvider.getMetaSettings().clear();
+    private void clearPrefs(Context context) {
+        settingsProvider.clearAll();
```

Comment:
I ran into problems where some state from the old version of the app would lie around an realized we needed some more aggressive housekeeping here.